research/lamp-automated-model-parallelism/README.md

```diff
@@ -0,0 +1,53 @@
+# LAMP: Large Deep Nets with Automated Model Parallelism for Image Segmentation
+
+<p>
+<img src="./fig/acc_speed_han_0_5hor.png" alt="LAMP on Head and Neck Dataset" width="500"/>
+</p>
+
+
+> If you use this work in your research, please cite the paper.
+
+A reimplementation of the LAMP system originally proposed by:
+
+Wentao Zhu, Can Zhao, Wenqi Li, Holger Roth, Ziyue Xu, and Daguang Xu (2020)
+"LAMP: Large Deep Nets with Automated Model Parallelism for Image Segmentation."
+MICCAI 2020 (Early Accept, paper link: https://arxiv.org/abs/2006.12575)
+
+
+## To run the demo:
+
+### Prerequisites
+- install the latest version of MONAI: `git clone https://github.com/Project-MONAI/MONAI` and `pip install -e .`
```

Comment:
Let's try to update to `pip install monai==0.2.0` later, otherwise if we don't maintain this research example, it may be not compatible with further MONAI versions.

Reply:
sounds good